task2/repository.js

```diff
@@ -29,7 +29,7 @@ const updateUser = (userId, updateData) => {
 
 const removeUser = (userId) => {
```

Comment:
This is a general comment for repository.js file. This is basically your data access object as generally refer as DAO. You can move this file directly into domain/user and maybe name it like user.service. Repository is also fine.